packages/block-library/src/navigation-menu/editor.scss

```diff
@@ -12,7 +12,88 @@
 }
 
 .wp-block-navigation-menu-item {
-	.wp-block-navigation-menu-item {
+	.wp-block-navigation-menu-link {
 		margin-left: $grid-size-large;
 	}
 }
+
+/**
+ * Colors Selector component
+ */
+$colors-selector-size: 22px;
+.block-editor-block-colors-selector {
+	width: auto;
+
+	// Toolbar colors-selector button.
+	.block-editor-block-colors-selector__toggle {
+		display: block;
+		margin: 0 auto;
+		padding: 3px;
+		width: auto;
+
+		// Button container.
+		.block-editor-block-colors-selector__icon-container {
+			width: 42px;
+			height: 30px;
+			position: relative;
+			margin: 0 auto;
+			padding: 3px;
+			display: flex;
+			align-items: center;
+			border-radius: 4px;
+
+			// colors-selector - selection status.
+			.colors-selector__state-selection {
```

Comment:
How necessary is this selector nesting. I've found it helpful to try and limit the _specificity_ of my selectors as it makes them easier to overide by others building on top of my work. 

So unless you absolutely need this much specificity here then I'd be inclined to avoid nesting selectors whereever possible.